br/pkg/lightning/restore/check_info.go

```diff
@@ -263,7 +315,7 @@ func (rc *Controller) checkRegionDistribution(ctx context.Context) error {
 }
 
 // CheckClusterRegion checks cluster if there are too many empty regions or region distribution is unbalanced.
-func (rc *Controller) CheckClusterRegion(ctx context.Context) error {
+func (rc *Controller) CheckClusterRegion(ctx context.Context, controller *pdutil.PdController) error {
```

Comment:
seems the new controller  argument is not used

Reply:
Yes.... It is a bad code...

Reply:
Done